core/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+from pure_pagination import EmptyPage
+from pure_pagination import Paginator
+from django.shortcuts import render
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
```

Comment:
Yeah that code create a memopol{vote,representative} automagically when a
vote or a representative is created, apparently luxcem didn't find to add a
one to one relation with {representative,vote}profile models and went for
smarter inheritance even if there's only one subclass of each ... I'm all
for simplification of this but I don't know how much this would delay the
release since it would introduce BC breaks. So what's our priority here ?
On Dec 19, 2015 3:13 PM, "Laurent Peuch" <notifications@github.com> wrote:

> In core/utils.py
> <https://github.com/political-memory/political_memory/pull/44#discussion_r48092711>
> :
>
> > @@ -0,0 +1,50 @@
> > +from __future__ import absolute_import
> > +
> > +from pure_pagination import EmptyPage
> > +from pure_pagination import Paginator
> > +from django.shortcuts import render
> > +
> > +
> > +def create_child_instance_from_parent(child_cls, parent_instance):
>
> Wasn't this way too magic code supposed to be gone?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/political-memory/political_memory/pull/44/files#r48092711>
> .
>